### PR TITLE
Jetpack Compose Material3 1.3.0-beta05

### PR DESCRIPTION
**Pull request by igordmn**
Copied from b15c04590b904b5e626f97976a19074a5d26a66b

### CODE REVIEW

File: compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Slider.kt
Comment (MatkovIvan) [inline compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Slider.kt:1]: Our `androidx.compose.material3.SliderTest` fails. Probably we need to copy the changes from Android one

Comment (igordmn) [reply compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/Slider.kt:1]: Fixed https://github.com/JetBrains/compose-multiplatform-core/pull/1479/commits/767a88e870433ec2bd2871620009e8260b97db93

File: compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/AccessibilityUtil.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import androidx.annotation.VisibleForTesting
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.layout
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.offset
+
+@VisibleForTesting internal val HorizontalSemanticsBoundsPadding: Dp = 10.dp
```
Comment (MatkovIvan) [inline compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/AccessibilityUtil.kt:28]: Does it work well on all platforms?

**Review by MatkovIvan — APPROVED** [review]

File: compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/AccessibilityUtil.kt (hunk 1 above)
Comment (igordmn) [reply compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/internal/AccessibilityUtil.kt:28]: I checked Desktop usual interaction (works the same) and VoiceOver borders (they are increased and looks better)

**Pull request merged**